XML/TestCases/Other.xml

```diff
@@ -76,4 +76,22 @@
       <SetupType>long-term</SetupType>
     </SetupConfig>
   </test>
+  <test>
+    <testName>NTTTCP-MONITOR-THROUGHPUT</testName>
+    <testScript>NTTTCP-MONITOR-THROUGHPUT.ps1</testScript>
+    <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\ntttcp_monitor_throughput.sh</files>
+    <Platform>Azure</Platform>
+    <TestParameters>
+      <param>seconds_of_runtime=NTTTCP_MONITOR_SECS_RUNTIME</param>
```

Comment:
Do you think those 3 parameters are changed frequently?

Reply:
yes, they can be configured in different testing scenario.